Tighten Introduction (#2)
YubicoLabs/cose-two-party-signing-algs-rfc: draft-lundberg-cose-two-party-signing-algs.md

Since different signature algorithms digest the message in different ways
and at different stages of the algorithm,
there is no generally-applicable way to define such a division point
for every possible signature algorithm.
Therefore, this document defines algorithm identifiers encoding,
for a specific set of signature algorithms,
which steps of the signature algorithm are performed by the _digester_ (e.g., software application)
and which are performed by the _signer_ (e.g., HSM).
In general, the _signer_ holds exclusive control of the signing private key.

Note that these algorithm identifiers do not define new "pre-hashed" variants of the base signature algorithm,
nor an intermediate "hash envelope" data structure, such as that defined in [COSE-Hash-Envelope].

> Rather, these are correspond to existing signature algorithms
> that would typically be executed by a single party,

Reviewer: Typo?
```suggestion
Rather, these identifiers correspond to existing signature algorithms
that would typically be executed by a single party,
```

Reviewer: Alternatively:
```suggestion
Rather, these identifiers represent existing signature algorithms
that would typically be executed by a single party,
```
